modules/iam/tagging.tf

```diff
@@ -31,7 +24,6 @@ locals {
     var.create_iam_resources,
     var.create_iam_tag_namespace,
     local.tag_namespace_id_found == null,
-    one(data.oci_identity_tags.oke[*].tags) == null,
```

Comment:
If the tag namespace id is found, then this looks for if the namespace has any tags in it, but the previous line/boolean already ensures that the tag namespace id should not be found, so this line never actually matters